model/src/main/java/com/ge/predix/acs/rest/AttributeConnector.java

```diff
@@ -2,9 +2,12 @@
 
 import java.util.Set;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class AttributeConnector {
     private boolean isActive;
     
+    @JsonProperty(defaultValue="480", required=false)
```

Comment:
I don't think this will actually work.. Jackson's core databind currently only uses `defaultValue` for documenting the expected default value of the property instead of actually enforcing it. If you want to keep this, we need to also change the actual assignment to use a default value like: `private int maxCachedIntervalMinutes = 480;`